components/apimgt/org.wso2.carbon.apimgt.rest.api.store/src/main/java/org/wso2/carbon/apimgt/rest/api/store/impl/ApisApiServiceImpl.java

```diff
@@ -55,14 +55,20 @@
      * @param ifUnmodifiedSince If-Unmodified-Since header value
      * @param request           msf4j request object
      * @return 200 response if the deletion was successful
-     * @throws NotFoundException If commment not found
+     * @throws NotFoundException
```

Comment:
Add the reason for the null NotFoundException. Check the whole PR. there are multiple places.